app/static/css/style.css

```diff
@@ -68,6 +101,10 @@ div.header {
   height: 50px;
 }
 
+.font-alt {
```

Comment:
This doesn't seem right. `font-alt` is supposed to switch to the font that the current element isn't using. For example, it used to go `Montserrat -> Nunito Sans` or `Nunito Sans -> Montserrat`. Options are to either override every occurance of this in Cirrus :-1: or just add clone Cirrus to the repository and replace all instances of the fonts with the other one :+1:.

Reply:
Turns out I may be wrong on this, please check the Cirrus source code.

Reply:
Yup, what they do with font-alt is just specify that it is nunito sans, not what you described above.

Line 98 of https://github.com/Spiderpig86/Cirrus/blob/master/src/core/font.scss 

I would just be overwriting Nunito Sans to be Poppins.

Reply:
Let me know if that's okay